queue/queue.py

```diff
@@ -10,30 +10,37 @@ def __init__(self):
         """
         Initialize the Queue with an empty LinkedList
         """
-        pass
+        super().__init__()
+        self.items = LinkedList()
 
     def is_empty(self):
         """
         :returns: True if the LinkedList is empty and False if it isn't
         """
-        pass
+        if not self.items.head:
+            return True
+        return False
```

Comment:
The line Raymond said you should add would basically be all that is in the function (minus the `not`). It's just a shorter way to write what you have. `self.items.head is None`  would be `true` if the head is None, otherwise, `false`.
```suggestion
        return self.items.head is None
```

Reply:
Thank you 
@jirevwe `is None` is ok

Reply:
Yeah, this is better